docs/micropython.md

```diff
@@ -0,0 +1,45 @@
+# Using python-ev3dev with MicroPython
+
+The core modules of this library are shipped as a module for [MicroPython](https://micropython.org/),
+which is faster to load and run on the EV3. If your app only requires functionality supported on
+MicroPython, we recommend you run your code with it for improved performance.
+
+## Module support
+
+```eval_rst
+==============================  =================
+Module                          Support status
+==============================  =================
+`ev3dev2.button`                ❌
+`ev3dev2.control` [1]_          ⚠️
```

Comment:
For the control part I created issues for the areas that need work:
- #636 
- #637 
- #638